admin-dev/themes/new-theme/js/app/pages/stock/components/overview/products-actions.vue

```diff
@@ -134,6 +135,9 @@
       onChange(event: Event): void {
         this.$store.dispatch('updateBulkEditQty', (<HTMLInputElement>event.target).value);
       },
+      onArrowClick(value: number) {
+        this.$store.dispatch('updateBulkEditQty', value);
+      },
```

Comment:
```suggestion
      onChange(value: number) {
        this.$store.dispatch('updateBulkEditQty', value);
      },
```
Maybe you can simply fix the initial onChange callback like this?

Reply:
I still think this extra event may be overkill Instead of adding a custom event in place of the native `change` event maybe fixing the existing callback `onChange` is enough

I may be wrong but at least this should be tried